target/linux/ramips/dts/mt7628an_joowin_jw-wr758ac_v1.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an_joowin_jw-wr758ac.dts"
+
+/ {
+	compatible = "joowin,jw-wr758ac,v1", "joowin,jw-wr758ac", "mediatek,mt7628an-soc";
```

Comment:
`"joowin,jw-wr758ac-v1"`

Reply:
And probably the second entry "joowin,jw-wr758ac" is useless and may be dropped (same below).

Reply:
Fixed in latest push.